Gruntfile.js

```diff
@@ -133,43 +133,36 @@ module.exports = function( grunt ) {
 
 					"requirejs/require.js": "requirejs/require.js",
 					"requirejs-domready/domReady.js": "requirejs-domready/domReady.js",
-					"requirejs-text/text.js": "requirejs-text/text.js",
+					"requirejs-text/text.js": "requirejs-text/text.js"
 				}
 			}
 		},
-		jshint: {
+		eslint: {
 			options: {
-				jshintrc: true
+
+				// See https://github.com/sindresorhus/grunt-eslint/issues/119
+				quiet: true,
+				configFile: ".eslintrc",
+				fix: true
 			},
-			all: {
-				src: [ files.source, files.grunt, files.karma, files.speed, files.tests ]
-			}
-		},
-		jscs: {
+
+			// We have to explicitly declare "src" property otherwise "newer"
+			// task wouldn't work properly :/
 			src: {
-				options: {
-					requireDotNotation: null
-				},
-				src: [ files.source ]
+				src: files.source // requireDotNotation: null
+			},
+			dev: {
+				src: [ "src/**/*.js", "Gruntfile.js", "test/**/*.js", "build/**/*.js" ]
```

Comment:
Why do you specify these paths instead of using previous versions, e.g.: `[ files.source, files.grunt, files.karma, files.speed, files.tests ]`?

Reply:
I have added the same configuration as [jQuery](https://github.com/jquery/jquery/blob/6ced2639b567ab4a5089f38a8e90efedc12801bf/Gruntfile.js#L99-L114) what do you think is the best approach?

Reply:
Changed.